### ISSUE TITLE
network.data messages not shown

### ISSUE THREAD

**Issue by schmurfy** [noflo/noflo-ui#600]
I thought it was my runtime but I just checked noflo-nodejs and I get the same result, `network.begingroup`  and `network.endgroup` both show in the debug panel when the edge is selected but the `network.data` is not, I tried this with the lastest noflo-ui stable release and master and I get the same result.

Is it a bug or do I need to do something for these to appear ?

Here is what the noflo-nodejs sent:

```json
{
  "protocol": "network",
  "command": "begingroup",
  "payload": {
    "id": "filesystem/ReadFile_vb059() OUT -> IN core/Output_p8j8a()",
    "graph": "main",
    "src": {
      "node": "filesystem/ReadFile_vb059",
      "port": "out"
    },
    "tgt": {
      "node": "core/Output_p8j8a",
      "port": "in"
    },
    "group": "/tmp/flo.txt"
  }
}

{
  "protocol": "network",
  "command": "data",
  "payload": {
    "id": "filesystem/ReadFile_vb059() OUT -> IN core/Output_p8j8a()",
    "graph": "main",
    "src": {
      "node": "filesystem/ReadFile_vb059",
      "port": "out"
    },
    "tgt": {
      "node": "core/Output_p8j8a",
      "port": "in"
    },
    "data": "yo\n"
  }
}

{
  "protocol": "network",
  "command": "endgroup",
  "payload": {
    "id": "filesystem/ReadFile_vb059() OUT -> IN core/Output_p8j8a()",
    "graph": "main",
    "src": {
      "node": "filesystem/ReadFile_vb059",
      "port": "out"
    },
    "tgt": {
      "node": "core/Output_p8j8a",
      "port": "in"
    },
    "group": "/tmp/flo.txt"
  }
}
```

And what I see is:
```
</tmp/flo.txt>
<//tmp/flo.txt>
```

**Comment by bergie** [comment]
Can confirm, not seeing data packets:

<img width="1284" alt="screen shot 2016-07-19 at 19 50 35" src="https://cloud.githubusercontent.com/assets/3346/16960521/2c0e3656-4dea-11e6-825a-7024899af450.png">

**Comment by bergie** [comment]
With noflo/noflo-runtime#88 we now get packet data as expected

<img width="816" alt="screen shot 2016-07-19 at 20 21 21" src="https://cloud.githubusercontent.com/assets/3346/16961622/9ce63d52-4dee-11e6-85d7-26cfa820640e.png">

That needs to be merged, released, and we need to bump the dep in noflo-ui

**Comment by bergie** [comment]
Fixed in 0.12.2

**Comment by schmurfy** [comment]
thanks :)
I tried to look at it but this project is really daunting, there are so many different libraries...
